src/components/Footer/Footer.js

```diff
@@ -1,9 +1,69 @@
-import "./Footer.css";
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
 
+import SearchIcon from '../../assets/search-blue.svg';
+/* import WatchlistIcon from '../../assets/watchlist-add.svg';
+ */
```

Comment:
```suggestion
```

Reply:
Worked on the watchlist page on the weekend while I was stuck. Will remove unused code and implement in the next user story.